# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libcap/hotfix-gcc-4.2.patch.avr32
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libcap-2.59/libcap/cap_alloc.c.vanilla	2021-10-07 20:02:03.980617403 +0200
+++ libcap-2.59/libcap/cap_alloc.c	2021-10-07 20:02:23.248618777 +0200
@@ -12,7 +12,7 @@
  */
 static cap_value_t _cap_max_bits;
 
-__attribute__((constructor (300))) static void _initialize_libcap(void)
+__attribute__((constructor)) static void _initialize_libcap(void)
 {
     if (_cap_max_bits) {
 	return;
--- libcap-2.59/libcap/cap_text.c.vanilla	2021-10-07 20:02:03.978617403 +0200
+++ libcap-2.59/libcap/cap_text.c	2021-10-07 20:02:14.832618177 +0200
@@ -677,7 +677,7 @@
  * If the constructor is called (see cap_alloc.c) then we'll need the
  * corresponding destructor.
  */
-__attribute__((destructor (300))) static void _cleanup_libcap(void)
+__attribute__((destructor)) static void _cleanup_libcap(void)
 {
     if (_cap_proc_dir == NULL) {
 	return;
